hotness/buildsys.py

```diff
@@ -96,8 +96,8 @@ def handle(self, package, upstream, version, rhbz):
             cmd = [
                 '/usr/bin/rpmdev-bumpspec',
                 '--new', upstream,
-                '-c', '"Latest upstream, %s for #%s"' % (upstream, rhbz),
-                '-u', '"%s"' % self.userstring,
+                '-c', 'Update to %s (#%s)' % (upstream, rhbz.bug_id),
```

Comment:
@ralphbean If you want to see how that works (it's really neat): https://www.youtube.com/watch?v=fVBck2Zngjo